public/docs/getting_started/compile.md

```diff
@@ -1,43 +1,59 @@
-# Location
+# Compiling contracts
 
-All of your contracts are located in the `./contracts` directory. By default you're given an example Solidity [contract](http://solidity.readthedocs.org/en/latest/contracts.html) file and an example Solidity [library](http://solidity.readthedocs.org/en/latest/contracts.html#libraries) file, both with filenames ending in `.sol`. Although a Solidity library is different than a contract, for documentation purposes we're going to lump these into the same word, "contracts".
+## Location
 
-# Command
+All of your contracts are located in your project's `contracts/` directory. As contracts are written in [Solidity](LINK), all files containing contracts will have a file extension of `.sol`. Associated Solidity [libraries](http://solidity.readthedocs.org/en/latest/contracts.html#libraries) will also have a `.sol` extension.
 
-To compile your contracts, simply run:
+With a bare Truffle [project](./project) (created through `truffle init`), you're given a single `Migrations.sol` file that is responsible for deploying your contracts to the blockchain. If you're using a [Truffle Box](/boxes), you will have multiple files here.
```

Comment:
This isn't quite true about the Migrations contract. The Migrations contract itself simply _records_ successful migrations; it doesn't do the deploying. The migrations feature in Truffle does the deploying, and that features uses the Migrations contract. 

This should probably be changed to something like: 
> you're given a single `Migrations.sol` file that helps in the deployment process.